src/components/Calculator.css

```diff
@@ -0,0 +1,49 @@
+
+.main-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100vh;
+  align-items: center;
+}
+
+table {
+  width: 500px;
+  height: 500px;
+  text-align: center;
+  background-color: #e0e0e0;
+}
```

Comment:
- [ ] I think it could be better if all your buttons have the same dimension, some of your buttons are small, others big and middle. It's better if all have the same dimension as in the provided design also for more esthetic 🙃 🙃 

Your design

<img width="439" alt="Screen Shot 2023-02-16 at 13 47 11" src="https://user-images.githubusercontent.com/74501523/219345812-8eec600b-a896-489d-9aa0-475d648a3f6a.png">

Provided design

<img width="333" alt="Screen Shot 2023-02-16 at 13 48 21" src="https://user-images.githubusercontent.com/74501523/219345882-8cbdd9d2-2692-40ed-9970-eb72c43d3640.png">